shared_kv_storage/kv_storage/remote_lmdb/shared_kv_dbstore.py

```diff
@@ -101,9 +102,10 @@ def get(self, table, key):
                 value = self._db_store.db_store_get(table, key)
             else:
                 value = None
-        except Exception as ex:
-            logger.error("Exception occurred in get - {}".
-                         format(ex))
+        except Exception:
+            # @TODO : Instead of suppressing exception here, pass it back
```

Comment:
Is db_store_get throw exception if key does not exists? If Yes then add comment.